### ISSUE TITLE
Time zone is lost

### ISSUE THREAD

**Issue by rasendubi** [heutagogy/heutagogy-backend#56]
Request:
```http
# Save bookmark
POST :url/bookmarks
Authorization: JWT :token
Content-Type: application/json

{
  "url": "http://github.com/heutagogy/heutagogy-backend",
  "title": "heutagogy/heutagogy-backend",
  "timestamp": "2016-11-19T01:31:15+0200",
  "read": "2017-01-20T01:42:12+0200"
}
```

Reply:
```json
{
  "id": 4,
  "read": "2017-01-20T01:42:12",
  "timestamp": "2016-11-19T01:31:15",
  "url": "http://github.com/heutagogy/heutagogy-backend",
  "title": "heutagogy/heutagogy-backend"
}
```

The timezone is just dropped. (Timestamps weren't even adjusted to UTC.)

The issue holds for updates, too.

**Comment by rasendubi** [comment]
`aniso8601` parses the timestamp correctly:
```
[nix-shell:~/prg/heutagogy-backend]$ python3
Python 3.5.2 (default, Jun 25 2016, 21:38:40) 
[GCC 5.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import aniso8601
>>> aniso8601.parse_datetime('2017-01-20T01:42:12+0200')
datetime.datetime(2017, 1, 20, 1, 42, 12, tzinfo=+2:00:00 UTC)
```

**Comment by rasendubi** [comment]
Provided a fix: #57.

**Comment by rasendubi** [comment]
Fixed by #57.
